### ISSUE TITLE
No CredentialsProvider has been registered

### ISSUE THREAD

**Issue by hatavan83** [ru-fix/gradle-release-plugin#3]
Hi @swarmshine 
I try run `createRelease` with -Pgit.login=<git.login> -Pgit.password=<git.password>, but it is not successful, the error is `Authentication is required but no CredentialsProvider has been registered`

My config in `gradle.properties` just have only `version = 0.0.1-SNAPSHOT`
I don't add `repositoryUrl, repositoryUser, repositoryPassword` because I think it is default and account is already put by command line  :)
Could you please give me more detail about this case?

**Comment by hatavan83** [comment]
@swarmshine 
 I try use with `-Pru.fix.gradle.release.login=<git.login> -Pru.fix.gradle.release.password=<git.password>` and then it pass `Authentication`,
But now it fail with valid `checkValidBranch` on CI because my HEAD is master (HEAD -> master, origin/releases/release-1.0)

```
Caused by: org.gradle.api.GradleException: Invalid release branch
	at ru.fix.gradle.release.plugin.BranchGardener.checkValidBranch(BranchGardener.kt:100)
	at ru.fix.gradle.release.plugin.BranchGardener.createRelease(BranchGardener.kt:53)
```
Can you have any solution for this case?

**Comment by swarmshine** [comment]
Hello, @hatavan83 
Thank you for feedback. Your case was reproduced and fixed in latest 1.3.16 release.
Also names of properties to provide authentication details changed to longer versions in order to minimize conflict with other plugins. You can find examples in project readme documentation.
